docs/reference/api/connectivity/networksocket/networksocket.md

```diff
@@ -76,43 +76,24 @@ Existing network interfaces:
 
 Here are example applications that are built on top of the network-socket API:
 
-* [HTTP and HTTPS](https://os.mbed.com/teams/sandbox/code/http-example/).
-* [MQTT](https://os.mbed.com/teams/mqtt/code/HelloMQTT/).
-* [CoAP](https://os.mbed.com/teams/sandbox/code/coap-example/).
-* [Websockets](https://os.mbed.com/cookbook/Websockets-Server).
-* [TCP ping-pong with a computer](https://github.com/armmbed/mbed-tcp-ping-pong).
-* [UDP ping-pong with a computer](https://github.com/armmbed/mbed-udp-ping-pong).
+- [HTTP and HTTPS](https://os.mbed.com/teams/sandbox/code/http-example/).
+- [MQTT](https://os.mbed.com/teams/mqtt/code/HelloMQTT/).
+- [CoAP](https://os.mbed.com/teams/sandbox/code/coap-example/).
+- [Websockets](https://os.mbed.com/cookbook/Websockets-Server).
+- [TCP ping-pong with a computer](https://github.com/armmbed/mbed-tcp-ping-pong).
+- [UDP ping-pong with a computer](https://github.com/armmbed/mbed-udp-ping-pong).
 
 ##### Example
 
 Here is an example of an HTTP client program. The program brings up Ethernet as the underlying network interface, and uses it to perform an HTTP transaction over a TCPSocket:
 
 [![View code](https://www.mbed.com/embed/?url=https://os.mbed.com/teams/mbed_example/code/TCPSocket_Example/)](https://os.mbed.com/teams/mbed_example/code/TCPSocket_Example/file/6b383744246e/main.cpp)
 
-#### Arm Mbed Mesh
-
-The Arm Mbed Mesh API allows the application to use the IPv6 mesh network topologies through the [Nanostack](/docs/v5.7/tutorials/mesh.html#nanostack) networking stack.
-
-Mbed OS provides two types of IPv6 based mesh networks:
-
-* 6LoWPAN_ND, loosely following the Zigbee-IP specification.
-* Thread, following the specification from Thread Group.
-
-Nanostack is the networking stack which provides both of these protocols. For more information on the stack internals, refer to [Nanostack documentation](/docs/v5.7/tutorials/mesh.html#nanostack). Application developers use Nanostack through Mbed Mesh API.
-
-The application can use the `LoWPANNDInterface` or `ThreadInterface` object for connecting to the mesh network and when successfully connected, the application can use the Mbed C++ socket APIs to create a socket to start communication with a remote peer.
-
-The `NanostackEthernetInterface` is provided for Ethernet.
-
-##### Supported mesh networking modes
-
-Currently, 6LoWPAN-ND (neighbor discovery) and Thread bootstrap modes are supported.
-
 #### Cellular
 
 The [CellularBase](/docs/v5.7/mbed-os-api-doxy/class_cellular_base.html) provides a C++ API for connecting to the internet over a Cellular device.
 
-Arm Mbed OS provides a [reference implementation of CellularBase](https://github.com/ARMmbed/mbed-os/tree/master/features/netsocket/cellular/generic_modem_driver).
+Arm Mbed OS provides a [reference implementation of CellularBase](https://github.com/ARMmbed/mbed-os/tree/master/features/cellular/easy_cellular).
```

Comment:
Query: Does this content exist in Doxyen? If so, I'll link there.

Reply:
Not yet because 5.8 content is not yet available in Doxygen

Reply:
@TeemuKultala Could you please use https://github.com/ARMmbed/mbed-os/pull/6280/files as an example to be sure the Doxygen generates?